cli/lib/kontena/cli/nodes/list_command.rb

```diff
@@ -38,6 +38,7 @@ def fields
         version: 'agent_version',
         status:  'status',
         initial: 'initial',
+        availability: 'availability',
```

Comment:
I think we should be careful not to add too many new columns to `node ls`... how about showing the status as `drained` (in yellow?) instead of `online` for these nodes? Offline nodes would still just be shown as offline.